### DISCUSSION TITLE
Release file name for `amd64` and `386` should use the real architecture

### DISCUSSION THREAD

**Discussion by felipeng** [aquasecurity/trivy#6187] [Ideas]
### Description

Why the `amd64` and `386` are the only architectures that don't use the real architecture in the release package name?

[Here](https://github.com/aquasecurity/trivy/blob/9c5e5a04ee691f0f9ad064bd0cedec3b317f6bd8/goreleaser.yml#L94)
```
...
    {{- if eq .Arch "amd64" }}64bit
      {{- else if eq .Arch "386" }}32bit
...
```

Example of Dockerfile for multi-arch-build, this is why is important to use the architecture name directly:
```
FROM alpine:3
RUN case $(uname -m) in \
        x86_64) ARCH=amd64; ;; \
        aarch64) ARCH=arm64; ;; \
        *) echo "Unsupported architecture"; exit 1; ;; \
    esac && \
    wget https://github.com/aquasecurity/trivy/releases/download/v0.49.1/trivy_0.49.1_Linux-${ARCH}.tar.gz
...
```

The Dockerfile works for `arm64` but not for `amd64` because the release file name is  `trivy_0.49.1_Linux-64bit.tar.gz` instead of `trivy_0.49.1_Linux-amd64.tar.gz`

### Target

Git Repository

### Scanner

None

**Comment by itaysk** [comment]
agree. I guess the intention originally was to make the filename user-friendlier, but in hindsight it might be better to leave it. having said that I don't think we will touch now. for your case, I see that you are customizing the `ARCH` anyway (e.g x86_64->amd64), so you could give it the "trivy name"

**Reply by felipeng** [reply]
> agree. I guess the intention originally was to make the filename user-friendlier, but in hindsight it might be better to leave it. 

right


>  I see that you are customizing the ARCH anyway (e.g x86_64->amd64)

Yes, but the `case` is used to identify the ARCH and then use in others places, for example:

```
    wget https://github.com/terraform-linters/tflint/releases/download/v${VERSION_TFLINT}/tflint_linux_${ARCH}.zip -O /tmp/tflint.zip && \
    wget https://github.com/Shopify/kubeaudit/releases/download/v${VERSION_KUBEAUDIT}/kubeaudit_${VERSION_KUBEAUDIT}_linux_${ARCH}.tar.gz -O /tmp/kubeaudit.tar.gz && \
```

So creating an special condition only for `trivy` is not ideal
